packages/server/migrations/201907100048002-templates.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  up: async function (queryInterface, Sequelize) {
+    try {
+      await queryInterface.createTable('Templates', {
+        id: {
+          allowNull: false,
+          primaryKey: true,
+          type: Sequelize.UUID
+        },
+        name: {
+          type: Sequelize.STRING(127),
```

Comment:
do you have any specific reason to make this 127? Is this much better than a regular ``Sequelize.STRING``?

Reply:
I know that default length in Sequelize.STRING is 255, it is too long for this reason. We just want to give readable names for templates, no more, this is why.